### PR TITLE
[FIX#323]: Adding announce for when user focuses on selection toolbox 

### LINKED ISSUE #323: Screen reader does not read that we have moved to the selection toolbar or the buttons within it
Screen reader does not read that vi have moved to the selection toolbar or the buttons within it

### PR DESCRIPTION
**Pull request by a-m-dev**
fix #323 

Based on @haxxmaxx work related to nebula selection, just a bit tweaks and modifying the test cases to support (better to say on tests perspective, to make sure that we are) announce while user focuses on selection toolbar!

### CODE REVIEW

File: src/locale/locales/en.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
     "value": "Showing {0} rows and {1} columns.",
     "comment": "number of rows and columns announced by the screen reader. (tew 210929)"
   },
+  "SNTable.Accessibility.SelectionToolbar": {
+    "value": "Selection Toolbar",
```
Comment (haxxmaxx) [inline src/locale/locales/en.json:7]:
```suggestion
    "value": "Moved to selection Toolbar",
```

File: src/table/utils/handle-accessibility.js [hunk 2]
```diff
@@ -69,12 +69,13 @@ export const handleFocusoutEvent = (evt, shouldRefocus, keyboard) => {
...
 };
 
-export const focusSelectionToolbar = (element, keyboard, last) => {
+export const focusSelectionToolbar = (element, keyboard, last, announce) => {
   const clientConfirmButton = element
     .closest('.qv-object-wrapper')
     ?.querySelector('.sel-toolbar-confirm')?.parentElement;
   if (clientConfirmButton) {
     clientConfirmButton.focus();
+    announce({ keys: 'SNTable.Accessibility.SelectionToolbar' });
```
Comment (haxxmaxx) [inline src/table/utils/handle-accessibility.js:78]: This should run in both cases, so put it before the if block

File: src/locale/locales/en.json (hunk 1 above)
Comment (a-m-dev) [reply src/locale/locales/en.json]: ✅ Fixed

File: src/table/utils/handle-accessibility.js (hunk 2 above)
Comment (a-m-dev) [reply src/table/utils/handle-accessibility.js]: ✅ Fixed

**Review by haxxmaxx — APPROVED** [review]

**Comment by haxxmaxx** [comment]
Wait to merge this until I have merged `crl/focus-sel-nebula`

**Comment by haxxmaxx** [comment]
Close this since we will not solve this inside the table

**Pull request closed without merging**